gateway/api/serializers.py

```diff
@@ -254,5 +260,6 @@ def update(self, instance, validated_data):
         instance.env_vars = validated_data.get("env_vars", {})
         instance.artifact = validated_data.get("artifact")
         instance.author = validated_data.get("author")
+        instance.image = validated_data.get("image")
```

Comment:
I see we are only `uploading` by now. Is it because we are not adding the support to run it yet in this PR?

In case we need to add support for this in `RunProgramSerializer` and `RunProgramModelSerializer`.

Reply:
That is a good point. Should we add run? Or we want to deprecate it overall? I think we are settling on `build` -> `upload` -> `run`. WDYT @Tansito ?

Reply:
Haha I didn't think about it so far. Mmm... I would leave this pull request as it is by now and let's discuss if we remove `run` and open another pull request for it. WDYT?